Src/WitsmlExplorer.Frontend/components/ContentViews/table/Panel.tsx

```diff
@@ -36,13 +39,14 @@ const Panel = (props: PanelProps) => {
     columns,
     expandableRows = false,
     downloadToCsvFileName = null,
-    stickyLeftColumns
+    stickyLeftColumns,
+    showGraph = false
```

Comment:
Given that `showGraph` is used by only one view, the related functionality should be removed from `Panel`, and the display graph button should be passed through `panelElements` instead.

Reply:
fixed

Reply:
later also removed that due to using switch on view instead of button